src/utils/StatsigLocalStorage.ts

```diff
@@ -20,7 +20,9 @@ export default class StatsigLocalStorage {
       window != null &&
       window.localStorage != null
     ) {
-      return window.localStorage.setItem(key, value);
+      try {
+        window.localStorage.setItem(key, value);
+      } catch (e) {}
     } else {
       this.fallbackSessionCache[key] = value;
```

Comment:
Should we not use fallbackSessionCache if there's an error? And I guess read from both